gsd-web/Jakefile

```diff
@@ -0,0 +1,9 @@
+let { task, desc } = require('jake');
+
+const fs = require('fs');
+
+desc('This is the default task.');
+task('setup', function () {
+  fs.copyFile('.env.sample', '.env', fs.constants.COPYFILE_EXCL, (err) => { if (err) console.log(err) })
+  console.log('Copied .env.sample to .env')
+});
```

Comment:
I'll be honest, I'd never heard of a Jakefile before this. It seems similar in nature as rake commands in Ruby though, and very useful.